File: modul-01/bhasa c/PRAK110-2310817310003-Mfaridsetiawan.c
#include <stdio.h>
#include <math.h>

int main(){
 printf("Diketahui : \n");
 int alas , tinggi , sisia, sisib, sisic;
 printf(" Alas = ");
 scanf("%d", &alas);

 printf(" tinggi = ");
 scanf("%d", &tinggi);

 sisia = tinggi;
 sisib = sqrt(pow(alas, 2) + pow(tinggi, 2));
 sisic = alas;

 int keliling = sisia + sisib + sisic;
 int luas = 0.5 * alas * tinggi;

 printf("Sisi A = %d cm \n",sisia);
 printf("Sisi B = %d cm \n",sisib);
 printf("Sisi C = %d cm \n",sisic);
 printf("keliling = %d cm \n",keliling);
 printf("luas = %d cm \n", luas);

 return 0;
    
}

File: modul-01/python/PRAK110-2310817310003-Mfaridsetiawan.py
import math

alas = int(input("Masukkan Alas: "))
tinggi = int(input("Maukkan Tinggi:"))

sisiA = tinggi
sisiB = int(math.sqrt(alas**2 + tinggi**2))
sisiC = alas

keliling = sisiA + sisiB + sisiC
luas = int(0.5 * alas * tinggi)

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")

print("Jawab :")
print(f"Sisi A = {sisiA}  cm")
print(f"Sisi b = {sisiB}  cm")
print(f"Sisi c = {sisiC}  cm")
print(f"Keliling = {keliling} cm")
print(f"luas = {luas} cm")